src/components/Tokens/TokenDetails/index.tsx

```diff
@@ -58,6 +62,11 @@ const TokenTitle = styled.div`
     display: inline;
   }
 `
+const DividerLine = styled(Hr)`
```

Comment:
not seeing a divider
![Screenshot 2023-09-25 at 2 10 24 PM](https://github.com/Uniswap/interface/assets/11512321/7a9046e0-3ee8-4f2f-a593-7d17cd603180)

Reply:
Added the divider with the spacing requested by design